docs/data/run-frontier-squid-container.md

```diff
@@ -0,0 +1,133 @@
+Running Frontier Squid in a Container
+=====================================
+
+Frontier Squid is a distribution of the well-known [squid HTTP caching
+proxy software](http://squid-cache.org) that is optimized for use with
+applications on the Worldwide LHC Computing Grid (WLCG). It has
+[many advantages](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Why_use_frontier_squid_instead_o)
+over regular squid for common grid applications, especially Frontier
+and CVMFS. The OSG distribution of frontier-squid is a straight rebuild of the
+upstream frontier-squid package for the convenience of OSG users.
+
+!!! tip
+    OSG recommends that all sites run a caching proxy for HTTP and HTTPS to help reduce bandwidth and improve
+    throughput.
+
+This document outlines how to run Frontier Squid in a Docker container.
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points (consulting [the Reference section below](#reference) as needed):
```

Comment:
There's nor reference section in this doc, which is causing the CI to fail: https://github.com/opensciencegrid/docs/pull/749/checks?check_run_id=1736329950#step:6:14

Reply:
dropping the reference to the non-existent section

Reply:
I'd actually prefer that we link to this reference section: https://opensciencegrid.org/docs/data/frontier-squid/#reference. Seems relevant.